scss/utils/helpers/_responsiveVarContext.scss

```diff
@@ -28,39 +43,6 @@ $responsiveVarMap--base: (
 	large:	20px
 );
 
-/// Creates a Sass context in which the `$base` variable is rendered
```

Comment:
This should be a major version bump. There is some usage of `responsiveVarContext--[var]` mixins in `mup-web`, and potentially some in `pro-web`.

Before merging, we should:
1. Bump the _major_ version number in `Makefile`
2. Add a note to the release notes in `README.md` for `3.X.X` describing the features removed